CHANGES.rst

```diff
@@ -12,6 +12,23 @@ Features
   documentation for more information about why this change was made.
   See https://github.com/Pylons/pyramid/pull/3413
 
+- It is now possible to control whether a route pattern contains a trailing
+  slash when it is composed with a route prefix using
+  ``config.include(..., route_prefix=...)`` or
+  ``with config.route_prefix_context(...)``. This can be done by specifying
+  an empty pattern and setting the new argument
+  ``inherit_slash=True``. For example::
```

Comment:
If possible, please use the following to explicitly set the Pygments lexer to Python:

```
  ... For example:

  .. code-block:: python
```

Reply:
We have a history of only using bare-minimum syntax in the changes file so that it always renders ok on PyPI. If you want to check and see if PyPI supports code-block extensions now be my guest but I'll leave it this way until then.

Reply:
It does. See https://raw.githubusercontent.com/Pylons/pylons-sphinx-themes/master/README.rst and https://pypi.org/project/pylons-sphinx-themes/

Reply:
sounds good, fixed